kernel/scheduler/src/lib.rs

```diff
@@ -53,15 +53,32 @@ pub fn init() -> Result<(), &'static str> {
         })
     }
 
-    #[cfg(not(target_arch = "x86_64"))] {
+    #[cfg(target_arch = "aarch64")] {
+        interrupts::enable_timer_interrupts(true, aarch64_timer_handler)
+    }
+
+    #[cfg(not(any(target_arch = "x86_64", target_arch = "aarch64")))] {
```

Comment:
we can remove this whole log statement if you now support preemptive task switching on aarch64. That was just meant to be a placeholder for you.